examples/cxx/publish_subscribe_dynamic_data/src/publisher.cpp

```diff
@@ -38,19 +38,19 @@ auto main() -> int {
                          .create()
```

Comment:
It's not from you, but could you rename the `worst_case_memory_size` to `expected_max_memory_size` from two lines further up?

Reply:
@elBoberido We will be removing it soon, is it worth the effort?

Reply:
@elBoberido Actually, with the other MR, this number is supposed to be number of slice elements, not the number of bytes. Changed to `maximum_elements`.